packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -468,8 +468,8 @@ const NIC_TYPE_OPTIONS = [
     .pick(
       createSelector(createGetObjectsOfType('USB_group'), usbGroups =>
         map(
-          filter(usbGroups, usbGroup => usbGroup.VUSB[0] === undefined),
-          usbGroup => usbGroup.PUSB
+          filter(usbGroups, usbGroup => usbGroup.VUSBs[0] === undefined),
+          usbGroup => usbGroup.PUSBs
```

Comment:
@b-Nollet Please check my changes make sense, I have not tested this.

Reply:
The changes do make sense, it avoids the _VM > advanced_ page to be replaced by `an error has occurred`